src/schnetpack/representation/weight_init.py

```diff
@@ -0,0 +1,64 @@
+import torch
+import numpy as np
+
+#generates a random square orthogonal matrix of dimension dim
+def square_orthogonal_matrix(dim=3, seed=None):
```

Comment:
this is a manual QR decomposition afaik. you can ust torch.qr directly and then use Q